package.json

```diff
@@ -22,7 +22,7 @@
   },
   "homepage": "https://github.com/dojo/grunt-dojo2-extras#readme",
   "devDependencies": {
-    "@dojo/loader": "^2.0.0-beta1.1",
+    "@dojo/loader": "./tarballs/dojo-loader-0.1.1-beta.1.tgz",
```

Comment:
We should be able to use node's loader for Intern 4 and ditch this dependency

Reply:
I've addressed this in a local commit and will push it soon